.woodpecker/docker.yml

```diff
@@ -0,0 +1,200 @@
+pipeline:
+  build-web:
+    image: node:16-alpine
+    commands:
+      - cd web/
+      - yarn install --frozen-lockfile
+      - yarn build
```

Comment:
@anbraten I wonder - do there exist flags for "release mode"  to optimize build or is this default?

Reply:
Build is already the production build with optimization. `yarn start` does the hot-reload dev build.

Reply:
We should definitely think about caching for the web builds `dist` folder & `node_modules`